core_lib/src/activeframepool.cpp

```diff
@@ -66,11 +66,11 @@ void ActiveFramePool::clear()
     mCacheFramesMap.clear();
 }
 
-void ActiveFramePool::resize(int n)
+void ActiveFramePool::resize(quint64 memoryBudget)
 {
-    n = std::min(n, 1500);
-    n = std::max(n, 10);
-    mMaxSize = static_cast<unsigned>(n);
+    memoryBudget = qMin(memoryBudget, quint64(1024) * 1024 * 1024 * 16); // 16GB
+    memoryBudget = qMax(memoryBudget, quint64(1024) * 1024 * 100); // 100MB
```

Comment:
These two lines can be combined with `qBound`

Reply:
Will do thanks